### PR TITLE
Admin merchants

### PR DESCRIPTION
**Pull request by reidsmiller**
Completed all user stories for Admin Merchants

### CODE REVIEW

**Review by AlejandroLopez1992 — APPROVED** [review]
the format_time_stamp at the end of top_selling_date is really neat. Is it an innate Rails method?

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -1,5 +1,47 @@
 class Admin::MerchantsController < ApplicationController
   def index
-    
+    @top_5_merchants = Merchant.top_5_merchants_by_total_revenue
```
Comment (JCBeldo) [inline app/controllers/admin/merchants_controller.rb:3]: Wow, this is interesting to see in the controller.  Would love to hear about this functionality.

File: app/models/invoice.rb [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+  enum status: ['cancelled', 'in progress', 'completed']
 
   def self.invoice_items_not_shipped
-    select("invoices.*").joins(:invoice_items).where(invoice_items: {status: ["pending", "packaged"]})
+    select('invoices.*').joins(:invoice_items).where(invoice_items: {status: ['pending', 'packaged']})
   end
 
   def format_time_stamp
-    created_at.strftime("%A, %B %e, %Y")
+    created_at.strftime('%A, %B %e, %Y')
```
Comment (JCBeldo) [inline app/models/invoice.rb:16]: NM

File: app/models/invoice.rb [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
   end
 
   def total_revenue
```
Comment (JCBeldo) [inline app/models/invoice.rb:19]: My PR/branch will consolidate this into one method.

File: app/models/item.rb [hunk 4]
```diff
@@ -3,7 +3,7 @@ class Item < ApplicationRecord
   belongs_to :merchant
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
-
+
```
Comment (JCBeldo) [inline app/models/item.rb:6]: Thinking to put this in a module?

**Review by JCBeldo — APPROVED** [review]
I foresee quite a few merge conflicts, but we'll get to it, no problem. Looks good.  Lots of hard methods in here.  Great job!

**Review by SMcPhee19 — APPROVED** [review]
Looks good to me gang.

**Pull request merged**